tools/integration-tests/tester/tests/testutil.go

```diff
@@ -397,6 +411,45 @@ func RequireBlocksAvailable(t *testing.T, nodes []*framework.Node, blockIDs map[
 	log.Println("Waiting for blocks... done")
 }
 
+// RequireBlocksOrphaned asserts that all nodes have received BlockIDs and marked them as orphaned in waitFor time, periodically checking each tick.
+func RequireBlocksOrphaned(t *testing.T, nodes []*framework.Node, blockIDs map[string]DataBlockSent, waitFor time.Duration, tick time.Duration) {
+	missing := make(map[identity.ID]map[string]struct{}, len(nodes))
```

Comment:
you can use types.empty or just an AdvancedSet with string?

Reply:
done